inc/taskjobview.class.php

```diff
@@ -159,6 +159,10 @@ public function ajaxGetForm($options)
     {
         $ID = $this->ajaxLoadItem($options);
         $this->showForm($ID, $options);
+       // hide taskjobs_list after a job has been selected from the taskjobs_list
+        echo Html::scriptBlock("$(document).ready(function() {
+            document.getElementById(\"taskjobs_list\").style.display =\"none\";
```

Comment:
```suggestion
           $("#taskjobs_list").hide();
```

Reply:
Ty, I followed your suggestion in the lastest commit but protecting the quote ;-)